sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -15585,7 +15587,8 @@ public void testOrderByAlongWithInternalScanQueryNoDistinct()
                         JoinType.INNER,
                         null,
                         ExprMacroTable.nil(),
-                        CalciteTests.createJoinableFactoryWrapper()
+                        CalciteTests.createJoinableFactoryWrapper(),
+                        JoinAlgorithm.BROADCAST
```

Comment:
I don't think there is meaning of this in this context as this engine doesn't support joinalgorithms ; so this change should be absent here or `null` ?

maybe use `JsonInclude.Include.NON_NULL` and make the class behave like `null` means `BROADCAST` ?

Reply:
Changed this as discussed